src/Broadcast.js

```diff
@@ -28,6 +28,26 @@ class Broadcast {
     });
   }
 
+  static atExcept(source, message, excludes = [], wrapWidth, useColor, formatter) {
```

Comment:
Should be fixed now, you're right that it is pretty goofy to have it be optional.